### ISSUE TITLE
Default model no longer seems coherent

### ISSUE THREAD

**Issue by bgreen-litl** [IndicoDataSolutions/finetune#173]
**Describe the bug**

It doesn't _seem_ like the default model is pretrained in the latest development branch.

In prior versions, the default model generated coherent text with `generate_text()` using a wide variety of seed words.  With the current default model I haven't been able to generate any coherent text at all. This includes seeding with many different words.  

I'm mostly trying to use this as a sanity check that things are working.  I don't mean that the generated text would need to be the same as prior versions, but this is giving me the impression that either the model is no longer pretrained or something went wrong in loading the model.  Is it still expected that the default model is pretrained?  

**Minimum Reproducible Example**

    >>> import finetune
    >>> finetune.__version__
    '0.5.9'

The current version outputs things along these lines, regardless of seed word:

    >>> from finetune import Classifier
    >>> model = Classifier()
    >>> model.generate_text('potatoes')
    `'_start_potatoes " \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n greyson \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n greyson \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n , \n \n \n \n \n greyson \n \n greyson greyson greyson greyson greyson greyson greyson greyson greyson \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n \n greyson \n \n \n \n \n \n \n \n '`

**Expected behavior**

    >>> import finetune
    >>> finetune.__version__
    '0.4.1'

This older version would generate text along these lines with a wide variety of seed words:

    >>> from finetune import Classifier
    >>> model = Classifier()
    >>> model.generate_text('potatoes')
    `'_start_potatoes , " she said . \n " i do n\'t know what you mean . " \n " you \'re the one who said you wanted to be a chef . " \n " i did ? " \n " yes . " \n " i do n\'t know what you \'re talking about . " \n " you do n\'t have to . " \n " i do n\'t ? " \n " no . " \n " i do n\'t ? " \n " no . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not ? " \n " because i do n\'t want to . " \n " why not '`

**Comment by madisonmay** [comment]
Hi @bgreen-litl, thanks for the sanity check, sorry I didn't notice this sooner. 

The 0.5.0 release involved a major refactor to migrate to tensorflow's estimator framework.  It's quite possible that during this process, the code for `generate_text` was migrated improperly.   

However, we ran a set of internal benchmarks to confirm that we could replicate previous model finetuning results on a set of a dozen datasets, so I think you'll find that if that is what you're using finetune for things will still function as intended.

Will try to get a patch for the `generate_text()` issue up as soon as possible!  Thanks again for the bug report.

--Madison

**Comment by benleetownsend** [comment]
Apologies for this, I have fixed this in #190. The following lines show an example output.

>>> from finetune import Classifier
>>> a = Classifier()
INFO:finetune: Visible Devices: {1: GeForce GTX 1080 Ti}
>>> a.generate_text()
'_start_\n " i do n\'t know , " i said . " i think i \'m going to go back to the hotel and get some sleep . " \n " you \'re not going to sleep ? ".....
